cylc/uiserver/data_store_mgr.py

```diff
@@ -72,6 +72,34 @@ def _inner(*args, **kwargs):  # works for serial & async calls
     return _inner
 
 
+def _call_to_tuple(fcn):
```

Comment:
@oliver-sanders : I've read the docs and think I might need you to explain this in person.

What are the limitations on the sort of functions I can use this on?

It certainly doesn't work on 

```python
def multiply(a, b):
    return a * b

def cube(a):
    return a ^ 3
```

If it's meant to be like this can we have some description in the docstring of what we can and can't input?